app/views/admin/take_part_pages/_form.html.erb

```diff
@@ -1,24 +1,92 @@
-<div class="row">
-  <div class="col-md-8">
-    <p class="warning">
-      Warning: changes to take part pages appears instantly on the live site.
-    </p>
-
-    <%= form_for take_part_page, url: [:admin, take_part_page] do |form| %>
-      <%= form.errors %>
-      <%= form.text_field :title %>
-      <%= form.text_area :summary, rows: 2, class: 'js-summary-length-counting', data: { 'count-message-threshold' => 160, 'count-message-selector' => '.summary-length-info' } %>
-      <div class="summary-length-info">Summary text should be 160 characters or fewer. <span class="count"></span></div>
-      <%= form.text_area :body, rows: 20, class: "previewable", required: true, data: {
-        module: "paste-html-to-govspeak"
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/warning_text", {
+      text: "Warning: changes to take part pages appears instantly on the live site."
+    } %>
+
+    <%= form_for take_part_page, url: [:admin, take_part_page], multipart: true do |form| %>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Title (required)"
+        },
+        name: "take_part_page[title]",
+        id: "take_part_page_title",
+        hint: "Title text should be 255 characters or fewer.",
+        heading_level: 2,
+        heading_size: "l",
+        value: form.object.title,
+        error_items: errors_for(form.object.errors, :title)
+      } %>
+
+      <%= render "govuk_publishing_components/components/textarea", {
+        label: {
+          text: "Summary (required)",
+          heading_level: 2,
+          heading_size: "l",
+        },
+        name: "take_part_page[summary]",
+        id: "take_part_page_summary",
+        rows: 2,
+        hint: "Summary text should be 255 characters or fewer.",
```

Comment:
i can see above it says `      <div class="summary-length-info">Summary text should be 160 characters or fewer. <span class="count"></span></div>
`

Should this be 160?

Reply:
> i can see above it says ` <div class="summary-length-info">Summary text should be 160 characters or fewer. <span class="count"></span></div>`
> 
> Should this be 160?

There's a few inconsistencies around this across Whitehall. I discussed this with @nnagewad yesterday: The validation when the form is submitted reports an error when the user inputs > 255 characters on both the summary and the title so we've added a hint for both to that effect (see comments above). It's something we should look at though - we haven't implemented a client-side validation here at all (as exists on bootstrap) for now because of these inconsistencies. Also 255 characters seems somewhat overlong for a title.

Reply:
Oh right apologies i missed that. LGTM then!